homeassistant/components/avri/sensor.py

```diff
@@ -0,0 +1,106 @@
+"""Support for Avri waste curbside collection pickup."""
+from datetime import timedelta
+import logging
+
+from avri.api import Avri, AvriException
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+CONF_COUNTRY_CODE = "country_code"
+CONF_POSTCODE = "postcode"
+CONF_HOUSE_NUMBER = "house_number"
+CONF_HOUSE_NUMBER_EXTENSION = "house_number_extension"
+DEFAULT_NAME = "avri"
+ICON = "mdi:trash-can-outline"
+SCAN_INTERVAL = timedelta(hours=4)
+DEFAULT_COUNTRY_CODE = "NL"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_POSTCODE): cv.string,
+        vol.Required(CONF_HOUSE_NUMBER): cv.string,
```

Comment:
Wouldn't it make more sense to use `int` in this case? Or are there house numbers that cannot be displayed in an `int`?

Furthermore, the example you give for the library is also using int:

```python
from avri.api import Avri
client = Avri('1234AB', 42)
```

```suggestion
        vol.Required(CONF_HOUSE_NUMBER): cv.positive_int,
```

Reply:
Good point! Didn't know there was a `positive_int` type. nice one.

Reply:
Did you intend to revert this with your latest commit?

Reply:
Ah, I did not. Think something went wrong whilst rebasing. Fixed it.

Reply:
Now I am uncertain whether or not the `house_number`, `house_number_extension` or both can only have a positive int as their value 😅

Reply:
Ah dang it... I meant to change `house_number`. Extension can be anything. I'm so sorry for the confusion. Fixed it for real this time!

Reply:
Thanks for the clarification!